integration-tests/tests/syscall_read.rs

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: Apache-2.0
+
+//! This crate contains integration tests, including the exit syscall payload test.
+
+#![deny(clippy::all)]
+#![deny(missing_docs)]
+
+mod common;
+use common::IntegrationTest;
+
+/// This test runs the read syscall payload in the SGX keep using the SGX shim.
+#[test]
+#[cfg_attr(not(any(has_sgx, has_sev)), ignore)]
+fn read() {
+    IntegrationTest::new("read").run(5, 0, "hello world\n", "h", "");
```

Comment:
If my comment above is accurate, the "h" here would turn into the full "hello world\n."

Reply:
It is, indeed, accurate.